### DISCUSSION TITLE
Biomass related questions

### DISCUSSION THREAD

**Discussion by wanyingw193** [NREL/ReEDS-2.0#104] [Q&A: Data]
Hi ReEDS Team, I have a few questions about the biomass supply curves in the model.

I was wondering about which scenario in the U.S. Billion Ton Update (DOE 2011) that was used to derive the biomass supply curve below?
![image](https://user-images.githubusercontent.com/78582696/156271013-10489332-62bc-416c-8e0b-6da115edf8a1.png)

"Biomass fuel prices are represented by a supply curve with five bins per region". Does this mean that the bins division are based on price differences? Do input biomass prices vary from year to year or static？

Which type of biomass is used in the model as a fuel input for dedicated biomass IGCC plants or coal-fired plants?

Thank you so much!

**Comment by bsergi** [comment]
Thanks for your message on this. We use the medium housing, low wood energy scenario for the biomass supply curves. This is the scenario that the Billion Ton Study considers as a baseline. 

You are correct about the binning--the Billion Ton Study takes all the biomass supply and then aggregates it to those specific price levels for each bin. We don't have variation in biomass prices per year, although if the model uses more biomass in one year then it may face more expensive resource costs from those higher-priced bins.

The only type of biomass we modeled is woody biomass; we exclude liquid fuels derived from crops or grasses. We have included some cost estimates for turning the woody biomass into pellets for combustion.

By the way, the latest version of ReEDS utilizes the more recent Billion Tons Study Data (2016). See below for an updated plot of the supply curve, now with some breakdowns by region.

![image](https://user-images.githubusercontent.com/6035236/156629791-adea6e69-195f-4b24-aa07-f612d712b030.png)

**Reply by wanyingw193** [reply]
Thank you for the explanation. I just want to confirm a few more points of confusions.

In the billion ton study, each scenario projects a different biomass price from 2014 to 2040, with different production densities and yields projected in each year. Which year in baseline scenario was chosen to plot the above biomass supply curve?

I may have missed some conversions here. In the [billion ton data table](https://bioenergykdf.net/bt16-2-download-tool/state?f%5B0%5D=scenario%3AMedium%20housing%2C%20low%20energy%20demands), the lowest biomass price is $30/dry ton, what cost conversion is needed to get the bioclass2 price of $10/dry ton in the supply curve of inputs\supplycurvedata\bio_supplycurve.csv?

Thank you in advance!

**Reply by bsergi** [reply]
For that supply curve data we used the 2040 projection. I'm not sure about your second question related to prices: the BTS reports supply curve at $10 increments which is why ours starts at $10/dry ton. I wasn't able to access the table you posted so I can't comment on why that data only shows down to $30/dry ton.

**Comment by wanyingw193** [comment]
Morning! Emissions from all biopowers are treated as -1* capture_rate(fraction)* emit_rate (tons per MMBtu) to reflect the net zero assumption. Based on the equation, biopower plant will have no emission, BECCS will have negative emission.
![image](https://user-images.githubusercontent.com/78582696/173291869-d6e38980-b6d7-4f8f-b495-c90d3ef03b13.png)

What is the reference source for this assumption? Thanks!

**Reply by bsergi** [reply]
Hi @wanyingw193 , my apologies for the long delay in response here. A couple of points here. First, BECCS is treated as negative emissions, but the amount of CO2 capture is based on the stack emissions when combusting woody biomass. That value (which is the value for "Biopower" found in `inputs/emitrate.csv`) is derived from Table 1.6-3 on page A4-12 of this report: https://www.nrel.gov/docs/fy03osti/33123.pdf (with some unit conversion). 

Emissions for biopower are then set to zero shortly after the line you quoted: `emit_rate_fuel(i,"CO2")$[sameas(i,"biopower")] = 0 ;
`. There isn't a source for this assumption; rather it is based on the understanding that all carbon in woody biomass would be taken from the atmosphere during the growth cycle. This of course misses any lifecycle CO2 associated with transport and storage, but ReEDS doesn't track upstream lifecycle emissions for other fuels so we wanted to be consistent here. This is likely an assumption we will revisit at some point in the future.
